db.py

```diff
@@ -19,7 +19,22 @@ def connect(self) -> sqlite3.Connection:
     def create_tables(self) -> None:
         tables = [
             """
-            CREATE TABLE IF NOT EXISTS users (
+            DROP TABLE IF EXISTS users;
+            """,
+            """
+            DROP TABLE IF EXISTS accounts;
+            """,
+            """
+            DROP TABLE IF EXISTS statements;
+            """,
+            """
+            DROP TABLE IF EXISTS categories;
+            """,
+            """
+            DROP TABLE IF EXISTS transactions;
+            """,
+            """
```

Comment:
I will update sql queries to the `CREATE TABLE IF NOT EXISTS`

Reply:
Actually, could you do this for me please, as a part of this MR

Reply:
Jus restore statements back, no need to drop tables as we might have existing data

Reply:
Updated